cmd/api-resources_test.go

```diff
@@ -64,11 +66,33 @@ func TestListObjectsV2Resources(t *testing.T) {
 			fetchOwner:   true,
 			maxKeys:      1000,
 			encodingType: "gzip",
+			errCode:      0,
```

Comment:
Maybe errCode can use values from `APIErrorCode` instead of integers. I will leave it up to the reviewers.

Reply:
I agree with @kannappanr .
We should replace `125` with `ErrIncorrectContinuationToken`.

Reply:
ok

Reply:
Hey @ebozduman  , the code for `ErrIncorrectContinuationToken` is `InvalidArgument` , Should i display that instead ??

Reply:
Hi @ebozduman  , 
Changed it to string pointing to the `APIErrorCode.Code` . 
So here for `ErrIncorrectContinuationToken`. It gives `InvalidArgument` as the errorCode

Reply:
please check